src/app.ts

```diff
@@ -20,6 +24,25 @@ app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+/** Parse Cookies */
+app.use(cookieParser());
+
+/** Session Configuration with Sequelize */
+app.use(
+  session({
+    secret: 'keyboard cat',
```

Comment:
this can be set for dev, but for prod this **needs** to be an environment variable otherwise our session authentication is trivially spoof-able